code/game/machinery/slotmachine.dm

```diff
@@ -55,44 +55,44 @@
 			
 /obj/machinery/slot_machine/proc/spin_slots(userName)
 	switch(rand(1,4050))
-		if(1)
+		if(1) // .02%
 			atom_say("JACKPOT! [userName] has won a MILLION CREDITS!")
 			event_announcement.Announce("Congratulations to [userName] on winning the Jackpot of ONE MILLION CREDITS!", "Jackpot Winner")
 			result = "JACKPOT! You win one million credits!"
 			resultlvl = "highlight"
 			win_money(1000000, 'sound/goonstation/misc/airraid_loop.ogg')
-		if(2 to 5)
+		if(2 to 5) // .07%
 			atom_say("Big Winner! [userName] has won a hundred thousand credits!")
 			event_announcement.Announce("Congratulations to [userName] on winning a hundred thousand credits!", "Big Winner")
 			result = "Big Winner! You win a hundred thousand credits!"
 			resultlvl = "good"
 			win_money(100000, 'sound/goonstation/misc/klaxon.ogg')
-		if(6 to 50)
+		if(6 to 50) // 1.08%
 			atom_say("Big Winner! [userName] has won ten thousand credits!")
 			result = "You win ten thousand credits!"
 			resultlvl = "good"
 			win_money(10000, 'sound/goonstation/misc/klaxon.ogg')
-		if(51 to 100)
+		if(51 to 100) // 1.21%
 			atom_say("Winner! [userName] has won a thousand credits!")
 			result = "You win a thousand credits!"
 			resultlvl = "good"
 			win_money(1000, 'sound/goonstation/misc/bell.ogg')
-		if(101 to 200)
+		if(101 to 200) // 2.44%
 			atom_say("Winner! [userName] has won a hundred credits!")
 			result = "You win a hundred credits!"
 			resultlvl = "good"
 			win_money(100, 'sound/goonstation/misc/bell.ogg')
-		if(201 to 300)
+		if(201 to 300) // 2.44%
 			atom_say("Winner! [userName] has won fifty credits!")
 			result = "You win fifty credits!"
 			resultlvl = "good"
 			win_money(50)
-		if(301 to 1000)
+		if(301 to 1000) // 17.26%
 			atom_say("Winner! [userName] has won ten credits!")
 			result = "You win ten credits!"
 			resultlvl = "good"
 			win_money(10)
-		else
+		else // 74.05%
```

Comment:
75.3%, 1000 to 4050 is 3050, divide that by 4050 and multiply by 100

Reply:
how did I mess that up 🤔